edx_sga/tasks.py

```diff
@@ -13,9 +16,37 @@
 from student.models import user_by_anonymous_id  # lint-amnesty, pylint: disable=import-error
 from opaque_keys.edx.locator import BlockUsageLocator
 
+from edx_sga.constants import ShowAnswer
+
 
 log = logging.getLogger(__name__)
 DATA_DIR = getattr(settings, "DATA_DIR", "/edx/app/edxapp/data")
+DEFAULT_FILE_STORAGE = getattr(settings, "DEFAULT_FILE_STORAGE", "django.core.files.storage.FileSystemStorage")
+
+
+def is_s3_default_storage():
+    """
+    returns true if s3 is default storage.
+    """
+    return True if DEFAULT_FILE_STORAGE and "s3boto" in DEFAULT_FILE_STORAGE else False
+
+
+def is_local_storage():
+    """
+    returns true if local storage is default storage.
+    """
+    return True if DEFAULT_FILE_STORAGE and "FileSystemStorage" in DEFAULT_FILE_STORAGE else False
+
+
+def _create_dir_local_storage(relative_destination_path):
+    """
+    create dir on local storage
+
+    Args:
+        relative_destination_path (str): submissions folder
+    """
+    if is_local_storage():
```

Comment:
Like I indicated in a different comment, we should not be calling a function called `_create_dir_local_storage` if we are not using local storage. Please move the conditional to the client(s) of this function, and if there's only one client, you should remove this function entirely